clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -1125,6 +1125,22 @@ void Clang::AddPreprocessingOptions(Compilation &C, const JobAction &JA,
     CmdArgs.push_back("__clang_openmp_device_functions.h");
   }
 
+  if (Args.hasArg(options::OPT_foffload_via_llvm)) {
+    // Add llvm_wrappers/* to our system include path.  This lets us wrap
+    // standard library headers and other headers.
+    SmallString<128> P(D.ResourceDir);
+    llvm::sys::path::append(P, "include");
+    llvm::sys::path::append(P, "llvm_offload_wrappers");
```

Comment:
I think append is variadic and you can do both pieces in one call

Reply:
Sorry, I pushed this first on top of old commits. This was fixed and I updated all my PRs.

Since I'm not able to do stacked commits, basically only review the latest one in each PR.